backend/app/controllers/users.js

```diff
@@ -126,9 +126,16 @@ module.exports = {
             var user = yield * insertOne(req, res, next);
 
             // Create user on Auth service
-            if (user) {
-                yield _createUserOnAuthService(req.body.email, req.body.password, req.body.roleID)
+            // TODO: https://jira.amida-tech.com/browse/INBA-609
+            var userAuthed = yield _getUserOnAuthService(req.body.email, req.headers.authorization);
```

Comment:
Can we create the auth user before creating the indaba user? Then we won't have to do a separate update to set the `authId`. I think we shouldn't be creating the Indaba user if we fail to create an auth service user anyway

Reply:
Will mark in 609.

Reply:
https://jira.amida-tech.com/browse/INBA-609